gradle/wrapper/gradle-wrapper.properties

```
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-5.6.2-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.1-bin.zip
```

Re-include the sources? See #733.

Theoretically you could also make this dependent on an environment variable (if that is even possible) to avoid the unnecessary additional download size if the sources are not used/wanted by the developer.

```suggestion
distributionUrl=https\://services.gradle.org/distributions/gradle-6.1-all.zip
```

---

We could, but I think this minimal optimization is not necessary.